src/UI/components/map/layers/drawerMap.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { useAppSelector } from '../../../state/hooks';
+import List from '@mui/material/List';
+import { styled } from '@mui/material/styles';
+import MuiDrawer from '@mui/material/Drawer';
+import Divider from '@mui/material/Divider';
+import IconButton from '@mui/material/IconButton';
+import MenuIcon from '@mui/icons-material/Menu';
+import ChevronLeftIcon from '@mui/icons-material/ChevronLeft';
+import { OverlayList } from './overlayList';
+import { BaseMapList } from './baseMapList';
+import {unpackOverlays} from './unpackOverlays';

```

Comment:
Can you fix this whitespace too please? In fact, can you add these lines to the `.eslintrc` file:
`   "object-curly-spacing": [1, "always", { "objectsInObjects": false }],
    "space-infix-ops": 1`